Task4.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import  { useEffect }from 'react'
+import  { useState } from 'react'
+
+function FetchData() {
+    const [records, setRecords] = useState([])
+
+    useEffect(() => {
+        fetch('https://jsonplaceholder.typicode.com/users')
+        .then(response => response.json())
+        .then(data => setRecords( data))
```

Comment:
I don't think we need a second then here line 11 is not required rather you can just extract data in line 10 and setRecords  ? Try it out.